api/MediaKeyMessageEvent.json

```diff
@@ -20,20 +20,32 @@
             "version_added": true
           },
           "ie": {
-            "version_added": null
+            "version_added": false
           },
           "opera": {
             "version_added": true
           },
           "opera_android": {
             "version_added": true
           },
-          "safari": {
-            "version_added": null
-          },
-          "safari_ios": {
-            "version_added": null
-          },
+          "safari": [
+            {
+              "version_added": "13.1"
```

Comment:
EME is pretty old and this doesn't seem right, http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Safari_12.0_OSX_10.13.6%22,%22Safari_12.0_OSX_10.14%22,%22Safari_12.1_OSX_10.13.6%22,%22Safari_12.1_OSX_10.14.4%22,%22Safari_13.0.2_OSX_10.15%22%5D&q=%22mediakeymessageevent%22 suggests it was exposed at least from 12.1, although seemingly depending on the OS version.

Can you double check?

Reply:
I don't fully trust the results for this, reverting the changes.